docs/pages/blog/mui-x-sep-2024-price-update.md

```diff
@@ -2,57 +2,74 @@
 
 ## Changes to the Pro plan
 
-### Current Pricing
+### Current pricing
 
-Under our current pricing model, if you licensed the Pro plan for 50 developers, you only paid for 10 seats. This concept was introduced when we offered only the Data Grid Pro with a much smaller feature set.
+Under the current pricing model, you only purchase a license for the first 10 developer.
+So to license the Pro plan for 50 developers you only need to purchase 10 licenses.
+This concept was introduced in the very beginning of the MUI X offering (2021) when we offered only the Data Grid Pro with a small feature set.
 
-### New Pricing
+Since the release of this component, we launched several new components and features, including [Date and Time Range Pickers](/x/react-date-pickers/date-time-range-picker/).
 
-With the launch of several new components, including [Date and Time Range Pickers](/x/react-date-pickers/date-time-range-picker/), as well as upcoming Pro versions of [Charts](/x/react-charts/) and [TreeView](/x/react-tree-view/), we're updating our pricing.
+### New pricing
 
-Moving forward, each developer requires a seat. So, for example, 50 developers will now need 50 seats.
+We will soon release the first Pro features of the [Charts](/x/react-charts/) and [Tree View](/x/react-tree-view/).
+These new features will only be available under the updated pricing: moving forward, each developer requires a license. So, for example, 50 developers will need 50 licenses.
 
-We're happy to discuss volume discounts to accommodate larger teams.
-
-### Grandfathering Current Customers
+### Grandfathering current customers
 
 We value our loyal customers and want to ensure a smooth transition: current customers will be grandfathered into their existing plans.
 
-This means you can continue using and renewing the legacy license for as long as you wish.
-However, the legacy license covers only Data Grid and Date Pickers.
+This means you can continue to renew your licenses under the legacy pricing for as long as you wish.
+However, the legacy pricing only covers the previous feature scope: Data Grid and Date Pickers.
 
 :::info
-**Access to new components:** The Pro versions of Charts and Tree View, and other future Pro components, will be available exclusively through a license key under the new Pro plan model or a Premium key (Premium users will experience no change).
+**Access to new components:** The Pro versions of Charts and Tree View, and other future Pro components, will be available exclusively through the new Pro plan model or a Premium plan (Premium customers will experience no change, all Pro components are included).
 ::::
 
-### Pricing for Small Teams
+### Pricing for small teams
+
+The price is the same for customers with 10 developers or fewer.
+Actually, we are making this pricing change to be able to keep the price affordable for small teams.
```

Comment:
Not sure it's super relevant, but it's pretty much why we do this, to lower price for smaller teams, and increase for larger team who get more value.